test/browser-compile/src/utils/css-template-plugin.ts

```diff
@@ -70,7 +70,7 @@ export const cssTemplatePlugin = {
   async load(id: string) {
     let code = styleImports.get(id.split('?')[0]);
     if (code) {
-      const results = await stencil.compile(code, {
+      const results = await stencil.transpile(code, {
```

Comment:
`compile` doesn't exist on the Stencil compiler API.

Reply:
Was this being masked by https://github.com/ionic-team/stencil/pull/3012#discussion_r693077180 clearing the console?

Reply:
This failed when running the start command in this directory, because compile didn't exist. It was a TypeScript error that was accurate, I'm assuming because it's been a while since this `browser-compile` project was started up. 

The comment you reference actually covered up errors in the worker, which if it wasn't here, would have immediately told me that how we get the fetch reference for the in-memory system was erroring.